packages/node_modules/@webex/plugin-meetings/src/meeting-info/util.js

```diff
@@ -198,7 +197,11 @@ MeetingInfoUtil.generateOptions = async (from) => {
     options.type = _CONVERSATION_URL_;
     try {
       await webex.internal.services.waitForCatalog('postauth');
-      const convoUrl = `${webex.internal.services.get('conversation')}/${CONVERSATIONS}/${hydraId.destination}`;
+
+      const convoUrl = webex.internal.conversation.getUrlFromClusterId({
+        cluster: hydraId.cluster,
+        id: hydraId.destination
+      });
```

Comment:
This logic appears untestable due to a lack of Webex object availability in the tests [common issue we see with testing these util methods]. This prevents stubbing of `getUrlFromClusterId()`.